packages/istanbul-lib-report/test/summarizer.test.js

```diff
@@ -321,6 +321,14 @@ describe('summarizer', function() {
             assert.deepEqual(nodes, ['g:']);
         });
 
+        it('handles getting root node name when empty coverage map', function() {
```

Comment:
my only minor nitpick here would be to describe the outcome. `handles` is good, but ideally this `does not crash` or something like that right?